src/Workspaces/CSharp/Portable/Extensions/StatementSyntaxExtensions.cs

```diff
@@ -13,6 +13,12 @@ namespace Microsoft.CodeAnalysis.CSharp.Extensions
 {
     internal static class StatementSyntaxExtensions
     {
+        public static bool IsFirstStatementInEnclosingBlock(this StatementSyntax statement)
+            => statement.Parent is BlockSyntax block && block.Statements.First() == statement;
+
+        public static bool IsFirstStatementInSwitchSection(this StatementSyntax statement)
+            => statement.Parent is SwitchSectionSyntax block && block.Statements.First() == statement;
```

Comment:
nit 'SwitchSectionSyntax switchSection`.

Reply:
@Neme12 do you have helpers for this already?  just askig before we check in duplicates.